src/app/pages/abandoned-animal/abandoned-animal.routes.ts

```diff
@@ -0,0 +1,21 @@
+import {Routes} from '@angular/router';
+import {RoutesUrlsEnum} from '@common/enums';
+
+/**
+ * Defines the routes for the abandoned animal module.
+ * @author gjimenez
+ */
+
```

Comment:
pegar mas los comentarios a la definicion de la clase

```suggestion
/**
 * Defines the routes for the abandoned animal module.
 * @author gjimenez
 */
export const ABANDONED_ANIMAL_ROUTES: Routes = [
```